appinventor/components/src/com/google/appinventor/components/runtime/Sound.java

```diff
@@ -298,13 +313,16 @@ public void run() {
   }
 
   private void playAndCheckResult() {
+    if (soundPool == null) {
```

Comment:
Did this happen in practice or is it just for extra protection?  In any case, maybe this should dispatch an error since it is unexpected that `soundPool` would be null here.

Reply:
Java problem 🙂 we never know